editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2009,6 +2011,17 @@ void TileSetAtlasSourceEditor::_tile_alternatives_control_mouse_exited() {
 	alternative_tiles_control_unscaled->queue_redraw();
 }
 
+void TileSetAtlasSourceEditor::_tile_alternatives_create_button_pressed(const Vector2i p_atlas_coords) {
+	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
+
+	// FIXME: doesn't undo changes to `next_alternative_id` counter
```

Comment:
```suggestion
	// FIXME: Doesn't undo changes to `next_alternative_id` counter.
```
It just means that some alternative IDs will be skipped when you undo and make them again, no?

Reply:
Correct. I mention it because the counter is saved, so a sequence of user actions "create tile"->"undo"->"save" produces an unexpected change in the resource file. Intuitive behavior would be to leave no evidence of undone change.